src/bindings/c/tests/ov_test.cpp

```diff
@@ -24,7 +24,7 @@ std::string plugins_xml_std = TestDataHelpers::generate_ieclass_xml_path("plugin
 #endif
 const char* plugins_xml = plugins_xml_std.c_str();
 
-std::map<ov_element_type_e, size_t> element_type_size_map = {{ov_element_type_e::BOOLEAN, 8},
+std::map<ov_element_type_e, size_t> element_type_size_map = {{ov_element_type_e::BL1, 1},
```

Comment:
Why did you rename element type?

Reply:
In windows it reports below error:
  include\10.0.18362.0\um\winnt.h(1131): error C2365: 'BOOLEAN': redefinition; previous definition was 'enumerator'
  openvino\src\bindings\c\include\openvino/c/ov_common.h(105): note: see declaration of 'BOOLEAN'

Reply:
Honestly I don't like that we need to rename data type. Do we have a stacktrace in order to find the place which causes the issue?

Reply:
I agree with you, I also don't like such rename behavior. But in C scope, there is no namespace to avoid such conflict with windows header files. Let me check whether there is other way to avoid renaming.

Reply:
I think also we need to understand the place which causes the issue

Reply:
ok, I am setup local windows build machine to check it.

Reply:
This conflict was caused by incorrect reference to windows.h header file, which has been removed already.  So we can restore its name.